build-system/tasks/release/index.js

```diff
@@ -139,6 +139,7 @@ function discoverDistFlavors_() {
     BASE_FLAVOR_CONFIG,
     ...Object.entries(experimentsConfig)
       .filter(
+        // Only include experiments that have a `define_experiment_constant` field.
         ([, experimentConfig]) => experimentConfig.define_experiment_constant
```

Comment:
Commenting to note it would be possible to write `([, {define_experiment_constant}]) => define_experiment_constant`, though I have no strong opinion on whether that would be better or worse.

Reply:
Because `define_experiment_constant` is not in camelCase the linter is complaining 😅
Punting...